browser-test/src/support/admin_questions.ts

```diff
@@ -25,6 +25,21 @@ export class AdminQuestions {
     expect(await this.page.innerText('h1')).toEqual('All Questions');
   }
 
+  async expectAdminQuestionsPageWithSuccessToast(successText: string) {
+    const toastContainer = await this.page.innerHTML('#toast-container');

```

Comment:
nit: Google typescript testing style guidelines recommend leaving an empty space between setting constants and the expect statements in a test. In other words, add an empty line between line 29 and 30.

Reply:
Oh nice, thanks! Updated here and in a couple of other spots in the file :)